full-moon/src/tokenizer/lexer.rs

```diff
@@ -650,6 +738,26 @@ impl Lexer {
             }
 
             '-' => {
```

Comment:
Maybe I'm missing it, but I don't see `..=`

Reply:
It is present earlier in the file https://github.com/Kampfkarren/full-moon/pull/276/files#diff-48f263edf3a28e94ca0dcbed94e5b2c8b0e1d87ca59ad1527218eb005c73cf22R708-R720